app/code/core/Maho/Captcha/etc/config.xml

```diff
@@ -0,0 +1,187 @@
+                </observers>
+            </controller_action_predispatch_newsletter_subscriber_new>
+            <controller_action_predispatch_review_product_post>
+                <observers>
+                    <captcha>
+                        <class>captcha/observer</class>
+                        <method>verify</method>
+                    </captcha>
+                </observers>
+            </controller_action_predispatch_review_product_post>
+            <controller_action_predispatch_wishlist_index_send>
+                <observers>
+                    <captcha>
+                        <class>captcha/observer</class>
+                        <method>verify</method>
+                    </captcha>
+                </observers>
+            </controller_action_predispatch_wishlist_index_send>
+        </events>
+        <layout>
+            <updates>
+                <captcha>
+                    <file>captcha.xml</file>
+                </captcha>
+            </updates>
+        </layout>
+        <translate>
+            <modules>
+                <Maho_Captcha>
+                    <files>
+                        <default>Maho_Captcha.csv</default>
+                    </files>
+                </Maho_Captcha>
+            </modules>
+        </translate>
+    </frontend>
+    <adminhtml>
+        <events>
+            <admin_user_authenticate_before>
+                <observers>
+                    <captcha>
+                        <class>captcha/observer</class>
+                        <method>verifyAdmin</method>
+                    </captcha>
+                </observers>
+            </admin_user_authenticate_before>
+            <controller_action_predispatch_adminhtml_index_forgotpassword>
+                <observers>
+                    <captcha>
+                        <class>captcha/observer</class>
+                        <method>verifyAdmin</method>
+                    </captcha>
+                </observers>
+            </controller_action_predispatch_adminhtml_index_forgotpassword>
+        </events>
+        <layout>
+            <updates>
+                <captcha>
+                    <file>captcha.xml</file>
+                </captcha>
+            </updates>
+        </layout>
+        <translate>
+            <modules>
+                <Maho_Captcha>
+                    <files>
+                        <default>Maho_Captcha.csv</default>
+                    </files>
+                </Maho_Captcha>
+            </modules>
+        </translate>
+    </adminhtml>
+    <crontab>
+        <jobs>
+            <captcha_cleanup>
+                <schedule><cron_expr>0 3 * * *</cron_expr></schedule>
+                <run><model>captcha/observer::cleanup</model></run>
+            </captcha_cleanup>
+        </jobs>
+    </crontab>
+    <default>
+        <admin>
+            <captcha>
+                <enabled>1</enabled>
+                <selectors>#page-login #loginForm
```

Comment:
I would prefer to have this line not customizable, backend selectors IMHO shouldn't be configurable because it's too easy to mess up, people would usually want to customize frontend stuff because they can change it, but backend to me shouldn't be messed with

Reply:
Yes, you're right it would be too easy to mess up. Commit a9d1ee7079f4ca3b6fe15df9a8b63b0c6a671e3e is perfect.